src/userRoutes.test.js

```diff
@@ -0,0 +1,82 @@
+jest.mock('../lib/users')
+jest.mock('../sequelize/db')
```

Comment:
your test file should match file name.

Reply:
test for `database_router.js` should be `database_router.test.js`

Reply:
name changed!